website/pages/docs/commands/volume/detach.mdx

```diff
@@ -0,0 +1,29 @@
+---
+layout: docs
+page_title: 'Commands: volume detach'
+sidebar_title: detach
+description: |
+  Detach volumes with CSI plugins.
+---
+
+# Command: volume detach
+
+The `volume detach` command detachs external storage volumes with Nomad's
+[Container Storage Interface (CSI)][csi] support. The volume must exist on the
+remote storage provider before it can be detached and used by a task.
```

Comment:
is this orphaned copy from the register page? It seems technically true but a little confusing here.

Reply:
Fair enough.